index.html

```diff
@@ -10,7 +10,379 @@
+                            <img class="main__team-socials-img" src="./images/twitter.svg" alt="instragram-icon" />
+                            <img class="main__team-socials-img" src="./images/facebook.svg" alt="twitter-icon" />
+                        </div>
+                    </div>
+                </div>
+                <div class="main__team-member-card">
+                    <div class="main__team-container-img">
+                        <img class="main__team-img" src="./images/cover (1).png" alt="employee" />
+                    </div>
+                    <div class="main__team-info">
+                        <div class="main__team-name">Rodney Stratton</div>
+                        <div class="main__team-job">Consultant</div>
+                        <div class="main__team-socials">
+                            <img class="main__team-socials-img" src="./images/instragram.svg" alt="facebook-icon" />
+                            <img class="main__team-socials-img" src="./images/twitter.svg" alt="instragram-icon" />
+                            <img class="main__team-socials-img" src="./images/facebook.svg" alt="twitter-icon" />
+                        </div>
+                    </div>
+                </div>
+                <div class="main__team-member-card">
+                    <div class="main__team-container-img">
+                        <img class="main__team-img" src="./images/cover (2).png" alt="employee" />
+                    </div>
+                    <div class="main__team-info">
+                        <div class="main__team-name">Avie Beaton</div>
+                        <div class="main__team-job">CO Founder</div>
+                        <div class="main__team-socials">
+                            <img class="main__team-socials-img" src="./images/instragram.svg" alt="facebook-icon" />
+                            <img class="main__team-socials-img" src="./images/twitter.svg" alt="instragram-icon" />
+                            <img class="main__team-socials-img" src="./images/facebook.svg" alt="twitter-icon" />
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </div>
+
+
+        <div class="main__contact">
+            <div class="main__contact-contents">
+                <div class="main__contact-sub-title">Contact us</div>
+                <div class="main__contact-title">Make an appointment</div>
+                <div class="main__contact-input-container">
+                    <div class="main__contact-input-container1">
+                        <input class="main__contact-name-input" type="text" placeholder="Full Name" />
+                        <input class="main__contact-email-input" type="text" placeholder="example@gmail.com" />
+                    </div>
+                    <div class="main__contact-layout-input-container1">
+                        <select class="contact-layout__input">
```

Comment:
Here also you are using a BLOCK which is not defined anywhere yet `contact-layout`